airflow/providers/docker/operators/docker.py

```diff
@@ -243,30 +245,39 @@ def _run_image(self):
                 tty=self.tty,
             )
 
-            lines = self.cli.attach(container=self.container['Id'],
```

Comment:
I think the [`self.cli.attach` method](https://docker-py.readthedocs.io/en/stable/api.html#docker.api.container.ContainerApiMixin.attach) with `stream=True` returns a generator which allows you to read the logs in a "streaming" fashion without using significant memory.

Reply:
> So I decided to just keep as a list to make the code less complex. And do not repeat the request as the original code.

I do read this in the description. I am guessing the original code makes the repeated request because of the tiny memory footprint of this API.